src/main/java/com/powsybl/openloadflow/ac/nr/NewtonRaphsonStoppingCriteriaType.java

```diff
@@ -0,0 +1,12 @@
+/**
+ * Copyright (c) 2019, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package com.powsybl.openloadflow.ac.nr;
+
+public enum NewtonRaphsonStoppingCriteriaType {
+    DEFAULT_CRITERIA,
+    CUSTOM_CRITERIA
```

Comment:
We have to find a proper name for those new stopping criteria, maybe @Hadrien-Godard has an idea ?

Reply:
DEFAULT_CRITERIA could be named UNIFORM_CRITERIA (and should be let as default type).
CUSTOM_CRITERIA could be named PER_EQUATION_TYPE_CRITERIA.